packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulator.jsx

```diff
@@ -22,7 +22,6 @@ const Accumulator = ({ accumulator_range_list, growth_rate, onChange, tick_size_
         <Fieldset
             className={classNames('trade-container__fieldset', 'accumulator')}
             header={localize('Accumulate')}
-            is_center
```

Comment:
If we don't pass `is_center` props, then `trade-container__fieldset-info--left` class will be applied. This class moves text for 5 px to the right.